backend/oasst_backend/utils.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+
+def get_detoxify_classification(inputs, hf_token):
+    API_URL = "https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta"
+    headers = {"Authorization": f"Bearer {hf_token}"}
+    payload = {"inputs": inputs}
+    response = requests.post(API_URL, headers=headers, json=payload)
```

Comment:
could you add a `response.raise_for_status()`?

Reply:
updated the pr

Reply:
@SzymonOzog thanks for the function .. unfortunately there has been a slight confusion .. there is already now merged a different version of this, see https://github.com/LAION-AI/Open-Assistant/blob/main/backend/oasst_backend/utils/hugging_face.py

.. but nevermind - not much time lost on this one.